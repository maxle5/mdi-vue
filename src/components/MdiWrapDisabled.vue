
  <template>
      <svg :width="size" :height="size" :viewBox="viewbox">
          <path d="M16,7H3V5H16V7M3,19H16V17H3V19M22,12L18,9V11H3V13H18V15L22,12Z" style="fill: currentColor" />
      </svg>
  </template>

  <script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiWrapDisabled",
      props: {
          size: { 
              type: Number, 
              default: 24 
          },
          viewbox: {
              type: String, 
              default: "0 0 24 24" 
          },
      },
  })
  </script>
