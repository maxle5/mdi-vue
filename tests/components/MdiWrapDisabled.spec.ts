
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiWrapDisabled from "../../src/components/MdiWrapDisabled.vue";

test("MdiWrapDisabled snapshot", () => {
  const wrapper = mount(MdiWrapDisabled, {});
  expect(wrapper.html()).toMatchSnapshot();
});
