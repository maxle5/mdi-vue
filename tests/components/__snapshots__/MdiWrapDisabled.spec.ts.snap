// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiWrapDisabled snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M16,7H3V5H16V7M3,19H16V17H3V19M22,12L18,9V11H3V13H18V15L22,12Z" style="fill: currentColor;"></path>
</svg>"
`;
